protobuf/benchmarks/compile-js.sh

```diff
@@ -10,7 +10,6 @@ OUTPUT_DIR="${SCRIPT_DIR}/temp"
 
 mkdir -p "${OUTPUT_DIR}"
 
-dart2js --trust-type-annotations        \
-        --trust-primitives              \
```

Comment:
So `-O4` enables `--trust-primitives`? I can't see this mentioned in [dart2js documentation](https://dart.dev/tools/dart2js).

Reply:
https://github.com/dart-lang/sdk/blob/40ad7aee40a82aba55a6a9481b9421c6785c9a2b/pkg/compiler/lib/src/options.dart#L787-L789